impl/src/main/java/com/okta/idx/sdk/impl/client/BaseIDXClient.java

```diff
@@ -478,14 +479,16 @@ public void revokeToken(String tokenType, String token) throws ProcessingExcepti
         try {
             Request request = new DefaultRequest(
                     HttpMethod.POST,
-                    clientConfiguration.getIssuer() + "/v1/revoke",
+                    isRootOrgIssuer(clientConfiguration.getIssuer()) ?
```

Comment:
We've got (almost) the same logic in 2 places now. I'm wondering if rather than `isRootOrgIssuer`, we should create a method that returns the normalized oauth path, which we could append the last part of the path to.

Reply:
Nice suggestion!